modules/tutorials/pages/tutorial-create-database.adoc

```diff
@@ -0,0 +1,74 @@
+= Create a Database
+:description: Create a database and use buckets, scopes, and collections to partition your data.
+:page-topic-type: tutorial
+:imagesdir: ../images
+:page-pagination: full
+:page-aliases: buckets-scopes-and-collections.adoc
+
+[abstract]
+{description}
+
+== Buckets, Scopes, and Collections
+
+To organize and manage your data in Couchbase, you can create buckets, scopes, and collections inside your cluster.
+
+A bucket is similar to a database, while scopes and collections are used to provide separation between documents of different types.
```

Comment:
Please see: https://docs.couchbase.com/cloud/clusters/data-service/about-buckets-scopes-collections.html

It's in the Capella doc, but that's the official breakdown of Bucket, Scope and Collection.

Reply:
Same limitations apply across Capella and Server on total number of scopes and collections, I'm pretty sure.